geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -167,6 +167,30 @@ public void sdiffWithDifferentyKeyType_returnsWrongTypeError() {
     assertThatThrownBy(() -> jedis.sdiff("ding")).hasMessageContaining(ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void sdiff_withDifferentKeyTypeAndTwoSetKeys_returnsWrongTypeError() {
```

Comment:
This test name might be a little clearer as something like "sdiff_withNonSetKeyAsFirstKey_returnsWrongTypeError"